tagls/server.py

```diff
@@ -286,7 +286,8 @@ async def initialize(ls: LanguageServer, params: types.InitializeParams):
         init_options: Dict[str, Any] = params.initialization_options
         show_message_log(f"initialization_options: {init_options}")
     else:
-        raise ValueError(f"Wrong initialization_options {params.initialization_options}")
+        init_options: Dict[str, Any] = {}
+        show_message_log(f"no initialization_options found, using default values")
```

Comment:
Thanks for your contribution! How about moving line 289 above 285 so that the type hinting of line 286 can be removed? And it might be better if:
```suggestion
        show_message_log(f"No initialization_options found, using default values")
```

Reply:
Updated~